### ISSUE TITLE
Does this project still use rc-table?

### ISSUE THREAD

**Issue by dandv** [cmogusu/my-site#1]

**Comment by cmogusu** [comment]
HI @dandv, unfortunately, this project does not use rc-tables

**Comment by dandv** [comment]
That's great. The problem is [it shows up in `yarn.lock`](https://github.com/cmogusu/my-site/search?q=rc-table&unscoped_q=rc-table), which makes GitHub think it does use `rc-table`, so this package shows somewhere among the [`rc-table` dependents](https://github.com/react-component/table/network/dependents?package_id=UGFja2FnZS0xNTA5OTgyOQ%3D%3D) (that's how I found it).

`yarn.lock` should be regenerated and comitted.

**Comment by cmogusu** [comment]
Yeah, you are right about that. I checked the yarn.lock file and found that one of the packages that I was using ([ant.d](https://www.npmjs.com/package/antd?activeTab=dependencies)) lists rc-table as a dependency. 
Would you still like me to try to remove the dependency? I am not sure that it's possible to do so without removing the [antd](https://www.npmjs.com/package/antd) package.
